modules/publicize/publicize-jetpack.php

```diff
@@ -673,10 +674,6 @@ function options_page_linkedin() {
 		Publicize_UI::options_page_other( 'linkedin' );
 	}
 
-	function options_page_google_plus() {
-		Publicize_UI::options_page_other( 'google_plus' );
```

Comment:
There are a lot of Google+ references in ui.php. Did you review those or are they slated to be removed in the far future?

Reply:
Strange… there is quite a bit more there :) I think I built this mostly from a .com patch that was mainly looking at API stuff. I'll take another look. Thanks!

Reply:
Removed more code in 2876529cef29284a503b896907fcc0b2ad87f524.